src/backend/base/langflow/base/textsplitters/model.py

```diff
@@ -1,15 +1,17 @@
 from abc import abstractmethod
 from typing import Any
 
+from langchain_core.documents import BaseDocumentTransformer
 from langchain_text_splitters import TextSplitter
 
+from langflow.base.document_transformers.model import LCDocumentTransformerComponent
 from langflow.custom import Component
 from langflow.io import Output
 from langflow.schema import Data
 from langflow.utils.util import build_loader_repr_from_data
 
 
-class LCTextSplitterComponent(Component):
+class LCTextSplitterComponent(LCDocumentTransformerComponent):
```

Comment:
Are there any risks to this change for other flows that use the LCTextSplitterComponent?

Reply:
I don't know. I don't see what could go wrong but...